### PR TITLE
Auto updated assets for terraform

### PR DESCRIPTION
**Pull request by yograjshisode**

### CODE REVIEW

**Comment by yograjshisode** [comment]
run jenkins-terraform-test

**Comment by yograjshisode** [comment]
run jenkins-terraform-test

**Review by mayank-avinetworks — APPROVED** [review]

File: avi/datasource_avi_inventoryfaultconfig.go [hunk 1]
```diff
@@ -5,28 +5,25 @@ package avi
 
 import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 
-func dataSourceAviJWTProfile() *schema.Resource {
+func dataSourceAviInventoryFaultConfig() *schema.Resource {
```
Comment (mayank-avinetworks) [inline avi/datasource_avi_inventoryfaultconfig.go:8]: Is this Overwriting `dataSourceAviJWTProfile` ?

Comment (yograjshisode) [reply avi/datasource_avi_inventoryfaultconfig.go:8]: No. JWTProfile is deprectated in 20.1.6.

**Pull request merged**